pyquil/noise.py

```diff
@@ -367,15 +368,14 @@ class NoisyGateUndefined(Exception):
     pass
 
 
-def get_noisy_gate(gate_name, params):
+def get_noisy_gate(gate_name: str, params: Tuple[float, ...]) -> Tuple[np.ndarray, str]:
```

Comment:
changing `Tuple[float, ...]` to `Iterable[ParameterDesignator]` fixes a couple more of the errors

Reply:
Fixed.